fizzArray.ts

```diff
@@ -0,0 +1,16 @@
+const fizzArray = n => {
```

Comment:
When using type I prefer writing functions like this fizzArray(input : <Type>).

Reply:
https://github.com/aaronshivers/angular-path/pull/7/commits/09536d7e54e31522356d4ffba3cd1ccaf06e8542